tools/scripts/seed/seedAuthUser.js

```diff
@@ -153,7 +153,18 @@ MongoClient.connect(MONGOHQ_URL, { useNewUrlParser: true }, (err, client) => {
   const db = client.db('freecodecamp');
   const user = db.collection('user');
 
+  const maybeDropWebhookTokens = async function () {
+    let collectionArr = await db
+      .listCollections({ name: 'WebhookToken' })
+      .toArray();
+
+    if (collectionArr.length > 0) {
+      await db.collection('WebhookToken').drop();
```

Comment:
Since we're dropping collections, could we log a warning and not seed anything unless the `FREECODECAMP_NODE_ENV=='development'`?

Bit paranoid, I know, but it seems best if this script just fails when run in production.

Reply:
FWIW, these scripts will never be run on staging and production.